btcpay-setup.sh

```diff
@@ -141,6 +140,17 @@ if [[ "$BTCPAYGEN_REVERSEPROXY" == "nginx" ]] && [[ "$BTCPAY_HOST" ]]; then
     BTCPAY_HOST="$DOMAIN_NAME"
 fi
 
+BTCPAY_CRYPTOS=""
+for i in $BTCPAYGEN_CRYPTO1 $BTCPAYGEN_CRYPTO2 $BTCPAYGEN_CRYPTO3 $BTCPAYGEN_CRYPTO4 $BTCPAYGEN_CRYPTO5 $BTCPAYGEN_CRYPTO5 $BTCPAYGEN_CRYPTO6 $BTCPAYGEN_CRYPTO7 $BTCPAYGEN_CRYPTO8
```

Comment:
should be in quote like `"$BTCPAYGEN_CRYPTO4"`

Reply:
done